### PR TITLE
Enable event logs and copy to Swift.

### PR DESCRIPTION
**Pull request by betobetodev**

### CODE REVIEW

File: application_manager/plugins/sahara/plugin.py [hunk 1]
```diff
@@ -532,7 +562,7 @@ def to_dict(self):
         }
 
     def execute(self, data):
-        app_id = "osspark" + self.id_generator.get_ID()
+        app_id = str(uuid.uuid4())[0:7]
```
Comment (armstrongmsg) [inline application_manager/plugins/sahara/plugin.py:583]: Is there a reason to use only the first eight characters as the "app_id"?

Comment (betobetodev) [reply application_manager/plugins/sahara/plugin.py:583]: No. Just seems to me a reasonable number of characters for an ID.

Comment (armstrongmsg) [reply application_manager/plugins/sahara/plugin.py:583]: ok, it's fine for me.

File: application_manager/utils/remote.py [hunk 2]
```diff
@@ -43,13 +43,13 @@ def read_file_from(remote_file, run_as_root=False, timeout=120):
 
 
 def execute_command(remote, key_path, command):
-    command = ('ssh -o "StrictHostKeyChecking no" -o "UserKnownHostsFile=/dev/null" -i %(key_path)s ubuntu@%(remote)s %(command)s' %
+    command = ('ssh -o "StrictHostKeyChecking no" -o "UserKnownHostsFile=/dev/null" -i %(key_path)s ubuntu@%(remote)s "%(command)s"' %
```
Comment (iurygregory) [inline application_manager/utils/remote.py]: Would be good make this to follow pep8 rules for the length of line characters

File: application_manager/utils/remote.py [hunk 3]
```diff
@@ -43,13 +43,13 @@ def read_file_from(remote_file, run_as_root=False, timeout=120):
...
                {'remote': remote, 'key_path': key_path, 'command': command})
 
     subprocess.call(command, shell=True)
 
 def execute_command_popen(remote, key_path, command):
-    command = ('ssh -o "StrictHostKeyChecking no" -o "UserKnownHostsFile=/dev/null" -i %(key_path)s ubuntu@%(remote)s %(command)s' %
+    command = ('ssh -o "StrictHostKeyChecking no" -o "UserKnownHostsFile=/dev/null" -i %(key_path)s ubuntu@%(remote)s "%(command)s"' %
```
Comment (iurygregory) [inline application_manager/utils/remote.py]: ditto

File: application_manager/utils/remote.py [hunk 4]
```diff
@@ -63,6 +63,13 @@ def copy_to_remote(remote, key_path, source, destination):
 
     subprocess.call(command, shell=True)
 
+def copy(key_path, source, destination):
+    command = ('scp -o "StrictHostKeyChecking no" -o "UserKnownHostsFile=/dev/null" -i %(key_path)s %(source)s %(destination)s' %
```
Comment (iurygregory) [inline application_manager/utils/remote.py]: ditto

**Review by iurygregory — CHANGES_REQUESTED** [review]

File: application_manager/utils/remote.py (hunk 2 above)
Comment (betobetodev) [reply application_manager/utils/remote.py]: Fixed.

File: application_manager/utils/remote.py (hunk 3 above)
Comment (betobetodev) [reply application_manager/utils/remote.py]: Fixed.

File: application_manager/utils/remote.py (hunk 4 above)
Comment (betobetodev) [reply application_manager/utils/remote.py]: Fixed.

**Pull request merged**